### PR TITLE
Adding configuration option to enable or disable websudo.

### PR DESCRIPTION
**Pull request by ronaldvanrij**
Adding configuration option to enable or disable websudo.

### CODE REVIEW

**Comment by igalic** [comment]
@ronaldvanrij something called web***sudo*** should definitely be documented.

File: manifests/init.pp [hunk 1]
```diff
@@ -44,6 +44,8 @@
   $gid          = undef,
   $shell        = '/bin/true',
 
+  $no_websudo   = false,
```
Comment (igalic) [inline manifests/init.pp]: this naming is absolutely confusing.

Comment (ronaldvanrij) [inline manifests/init.pp]: Agreed on the confusing parameter, unfortunately this reversed approach has been decided on by Atlassian, see the configuration item "jira.websudo.is.disabled". Reversing their decision by naming this parameter "websudo" may cause confusion as well. Nevertheless I'll update the parameter to a positive approach to controlling this feature.

Comment (juniorsysadmin) [inline manifests/init.pp]: Mirroring the the ugliness of the original variable is probably the better option: eg. ```$jira_websudo_is_disabled```

Comment (igalic) [inline manifests/init.pp]: software is sadness, made by people for other people.

Comment (JimPanic) [inline manifests/init.pp]: The Jira documentation says it is this: "Disabling Secure Administrator Sessions"; why not make it `$disable_secure_admin_sessions = false`?

Comment (JimPanic) [inline manifests/init.pp]: (`websudo` *is* a bad name)

Comment (igalic) [inline manifests/init.pp]: because *disable*_secure_admin_sessions, again, is confusing ;)

Comment (JimPanic) [inline manifests/init.pp]: Uhm. Okay. Then `$enable_secure_admin_sessions = true`. Both better than `anything_websudo`

Comment (igalic) [inline manifests/init.pp]: :+1:

**Comment by ronaldvanrij** [comment]
I've renamed the parameter to "enable_secure_admin_sessions". Added tests and updated Readme.MD.

**Comment by JimPanic** [comment]
<3

**Comment by igalic** [comment]
thank you so much @ronaldvanrij!

**Pull request merged**